internal/cli/cmd/plugin/build/build.go

```diff
@@ -120,11 +122,13 @@ func (o *option) Execute() error {
 
 	// If the schema is required, we need to prepare the CUE files for the archive
 	if o.isSchemaRequired {
-		if restoreInitialState, err := o.prepareCueFiles(); err != nil {
-			return fmt.Errorf("unable to prepare the CUE files for the archive: %w", err)
-		} else if restoreInitialState != nil {
+		restoreInitialState, cueErr := o.prepareCueFiles()
```

Comment:
this change is convering the fact even if there is an issue when vendoring the dependencies, we are still removing any backup folder created.